KIPs/kip-draft_kgp6.md

```diff
@@ -0,0 +1,672 @@
+---
+kip: 103
+title: Treasury Fund Rebalancing
+author: Aidan<aidan.kwon@klaytn.foundation>, Toniya<toniya.sun@klaytn.foundation>
+discussions-to: https://govforum.klaytn.foundation/t/kgp-6-proposal-to-establish-a-sustainable-and-verifiable-klay-token-economy/157
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-02-24
+---
+
+## Simple Summary
+
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the KIP.-->
+
+This proposal suggests the creation of a smart contract that records the rebalance of treasury funds. The main objective is to facilitate the approval and redistribution of treasury funds to new addresses while keeping record of the sender and receiver details.
```

Comment:
```suggestion
This proposal suggests a smart contract interface standard that records the rebalance of treasury funds. The main objective is to facilitate the approval and redistribution of treasury funds to new addresses while keeping record of the sender and receiver details.
```

Reply:
Yes, core part will be added briefly. There are some validation that can cancel the rebalancing, so everyone should know about the detail. I will update it soon with implementation of Klaytn repo.